CHANGELOG.md

```diff
@@ -80,6 +81,7 @@ All notable changes to this project will be documented in this file.
 -   \[INTERNALS\] Allow the use of magic methods to describe container methods.
 -   \[DEVELOPER\] Added an improved traceback to the developer-mode errors for errors in function calls.
 -   \[DEVELOPER\] Added an environment variable to globally activate developer-mode for errors.
+-   \[INTERNALS\] Check out the master branch of STC.
```

Comment:
This seems to be in conflict with this line:
https://github.com/pyccel/pyccel/blob/bbff872ce78c57d2c91d1895ca154ba8fe3b7d1a/CHANGELOG.md?plain=1#L8

It will also become outdated very quickly (as soon as there is a push in STC). Can you be more specific

Reply:
It is not possible to track a branch with git submodules. You can only pin to a specific commit. That is why I say:
> It will also become outdated very quickly (as soon as there is a push in STC)

I think we can still mention v5, but not v5.0 as this is associated with a specific tag (which we do not use after this PR).

There are 2 problems currently:
1. There are 2 messages in the CHANGELOG which imply 2 different versions of STC. Which are we actually using?
2. The new phrase is inaccurate.

Further I think this message is relevant for users as it can be seen in the generated code. It is therefore not internals.

I suggest updating the existing line:
> Ensure STC v5.0 is installed with Pyccel. 

to:
> Ensure STC >v5 (<COMMIT_SHA>) is installed with Pyccel.

Reply:
Got it!, [this](https://github.com/stclib/STC/commit/ef322aeb21c682a02253ce6df524307d142294c7) is the PR that added the reverse functions